Makefile

```diff
@@ -151,6 +151,9 @@ endif
 CAL_TESTING_KEY:=0
 ifneq ($(CAL_TESTING_KEY),0)
 DEFINES += HAVE_CAL_TESTING_KEY
+ifneq ($(CAL_TESTING_KEY),1)
+DEFINES += HAVE_CAL_TESTING_KEY_CI
+endif
 endif
```

Comment:
Plus clair je trouve:
```
CAL_TESTING_KEY:=0
CAL_TESTING_KEY_CI:=0
ifneq ($(CAL_TESTING_KEY),0)
DEFINES += HAVE_CAL_TESTING_KEY
endif
ifneq ($(CAL_TESTING_KEY_CI),0)
DEFINES += HAVE_CAL_TESTING_KEY_CI
endif
```

Dans le code .c
#if defined (HAVE_CAL_TESTING_KEY) && defined(HAVE_CAL_TESTING_KEY_CI)
#error "CAL KEY contradiction"
#endif

Reply:
Indeed, should be better now :+1: